src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -240,6 +240,7 @@ public function testRenderProvider()
                 array('ISBN', 'Title', 'Author'),
                 array(
                     array('<strong>99921-58-10-700</strong>', '<f>Divine Com</f>', 'Dante Alighieri'),
+                    array(new TableCell('<strong>99921-58-10-700</strong>'), new TableCell('<f>Divine Com</f>'), 'Dante Alighieri'),
```

Comment:
just added test in OutputFormatterTest and I think it should be merged into `2.3`.